packages/admin-stories/src/admin/toolbar/samples/ToolbarBreadcrumbs.tsx

```diff
@@ -0,0 +1,336 @@
+    ToolbarBreadcrumbs,
+    useStackSwitch,
+} from "@comet/admin";
+import { Box, Button, Container, Typography } from "@material-ui/core";
+import { TreeItem, TreeView } from "@material-ui/lab";
+import * as React from "react";
+
+const BreadcrumbTree = () => {
+    return (
+        <TreeView expanded={["1", "2", "3", "4"]} defaultCollapseIcon={<div>-</div>} defaultExpandIcon={<div>-</div>} defaultEndIcon={<div>-</div>}>
+            <TreeItem nodeId="1" label="Page1">
+                <TreeItem nodeId="2" label="Page 2">
+                    <TreeItem nodeId="2-1" label="Page 2-1" />
+                    <TreeItem nodeId="2-2" label="Page 2-2" />
+                </TreeItem>
+                <TreeItem nodeId="3" label="Page 3">
+                    <TreeItem nodeId="3-1" label="Page 3-1" />
+                    <TreeItem nodeId="3-2" label="Page 3-2" />
+                </TreeItem>
+                <TreeItem nodeId={"4"} label={"Page 4"} />
+            </TreeItem>
+        </TreeView>
+    );
+};
+const ToolbarBreadcrumb = () => {
+    return (
+        <StackApiContext.Consumer>
+            {(stackApi: IStackApi | undefined) => <ToolbarBreadcrumbs pages={stackApi?.breadCrumbs ?? []} />}
+        </StackApiContext.Consumer>
+    );
+};
+const Page1 = () => {
+    return (
+        <>
+            <Toolbar
+                actionItems={(stackSwitchApi: IStackSwitchApi | undefined) => {
+                    return (
+                        <>
+                            <Button
+                                variant={"contained"}
+                                color={"primary"}
+                                onClick={() => {
+                                    stackSwitchApi?.activatePage("page2", "foo");
+                                }}
+                            >
+                                <Typography>Go to Page 2</Typography>
+                            </Button>
+                            <Button
+                                variant={"contained"}
+                                color={"primary"}
+                                onClick={() => {
+                                    stackSwitchApi?.activatePage("page3", "foo");
+                                }}
+                            >
+                                <Typography>Go to Page 3</Typography>
+                            </Button>
+                            <Button
+                                variant={"contained"}
+                                color={"primary"}
+                                onClick={() => {
+                                    stackSwitchApi?.activatePage("page4", "foo");
+                                }}
+                            >
+                                <Typography>Go to Page 4</Typography>
+                            </Button>
+                        </>
+                    );
+                }}
+            >
+                <ToolbarBackButton />
+                <ToolbarBreadcrumb />
```

Comment:
Shouldn't this be called `<ToolbarBreadcrumbs />`?

Reply:
thats a bit confusing thats true, thats just a simple helper, maybe i move the fetching of the breadcrumbs into the `<ToolbarBreadcrumbs />` component